File: 23_C#Basics/_03_ComplexConditionalStatementsMoreExercises/_10_MultiplyBy2/_10_MultiplyBy2.cs
using System;

public class _10_MultiplyBy2
{
    public static void Main(string[] args)
    {
        // Read input value
        double number = double.Parse(Console.ReadLine());

        // Multiply the number by 2 and print while it's non-negative
        while (number >= 0)
        {
            double result = number * 2;
            Console.WriteLine($"Result: {result:F2}");

            number = double.Parse(Console.ReadLine());
        }

        // End the program if the number is below 0
        if (number < 0)
        {
            Console.WriteLine("Negative number!");
        }
    }
}

File: 01_ProgrammingBasics/_03_ComplexConditionalStatementsMoreExercises/_10_MultiplyBy2.java
package _01_ProgrammingBasics._03_ComplexConditionalStatementsMoreExercises;

import java.util.Scanner;

public class _10_MultiplyBy2 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        double number = Double.parseDouble(scan.nextLine());

        // Multiply the number by 2 and print
        while (number >= 0) {
            double result = number * 2;

            System.out.printf("Result: %.2f%n", result);

            number = Double.parseDouble(scan.nextLine());
        }

        // End the program if the number is below 0
        if (number < 0) {
            System.out.println("Negative number!");
        }

        scan.close();
    }
}
